src/services.cpp

```diff
@@ -56,14 +56,26 @@ bool MapperClass::MapInflation(pensa_msgs::SetFloat::Request &req,
     return true;
 }
 
-bool MapperClass::ResetMap(std_srvs::Trigger::Request &req,
-                           std_srvs::Trigger::Response &res) {
+bool MapperClass::ResetMap(pensa_msgs::SetFloat::Request &req,
+                           pensa_msgs::SetFloat::Response &res) {
     mutexes_.octomap.lock();
         globals_.octomap.ResetMap();
+        globals_.octomap.SetMemory(req.data);
+    mutexes_.octomap.unlock();
+
+    res.success = true;
+    return true;
+}
+
+bool MapperClass::InitializeMapToPathPlanningConfig(std_srvs::Trigger::Request &req,
```

Comment:
reset the map and load no-fly-zones as occupied, as well as fly-zones as free areas